mediator/src/controllers/service-request.ts

```diff
@@ -1,16 +1,19 @@
+import axios from 'axios';
 import {logger} from '../../logger';
+import {FHIR, CHT} from '../../config';
```

Comment:
There are some formatting differences, and I assume they would be fixed via #43 ?